dist/marionette.gridview.js

```diff
@@ -20,8 +25,8 @@
       name:     DEFAULT_WIDGET_NAME,
       x:        0,
       y:        0,
-      width:    0,
-      height:   0,
+      width:    1,
```

Comment:
Gridstack units, if you init a gridstack with a size of 5x5, you could place 5 widgets with widths of 1